Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:8
```

Comment:
Hvorfor præcis node 8, og ikke en noget nyere version?

Der mangler også en EXPOSE, som angiver hvad for en port applikationen kører på.